Sources/CreateXCFramework/Zipper.swift

```diff
@@ -94,9 +103,9 @@ struct Zipper {
 
 #if swift(>=5.5)
 private extension ResolvedPackage {
-    var packageName: String {
-        self.manifestName
-    }
```

Comment:
Any reason why these lines were commented? If no longer needed, the `#if swift(>=5.5)` condition shouldn't be needed either.